packages/date-parser/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@holistics/date-parser",
-  "version": "2.10.0",
+  "version": "2.11.0",
```

Comment:
should bump to major version 3 due to breaking changes

Reply:
yo

Reply:
Fixed: https://github.com/holistics/js/pull/12/commits/fc2c20e3e915eda5ca5e17bc9f52bb3aa752158d